Kickstarter-iOS/Views/Cells/ThanksCategoryCell.swift

```diff
@@ -3,14 +3,48 @@ import Library
 import Prelude
 import UIKit
 
+internal protocol ThanksCategoryCellDelegate: AnyObject {
+  func thanksSeeAllProjectsTapped(_ cell: ThanksCategoryCell, category: KsApi.Category)
+}
+
 internal final class ThanksCategoryCell: UITableViewCell, ValueCell {
-  @IBOutlet fileprivate var seeAllProjectsButton: UIButton!
+  internal weak var delegate: ThanksCategoryCellDelegate?
+  fileprivate let viewModel: ThanksCategoryCellViewModelType = ThanksCategoryCellViewModel()
+
+  @IBOutlet fileprivate var seeAllProjectCategoryButton: UIButton!
```

Comment:
Some places in this file we use (see all) `project` singular and other places `projects` plural. Would be nice to just have one convention.

Also is the name `seeAllProjectsButton` not more accurate here? Or perhaps `seeAllProjectsInCategoryButton` (feels a bit long)?